docs/src/components/Navigation.js

```diff
@@ -13,72 +14,50 @@ const isModifiedEvent = event =>
 
 const components = Object.keys(routes);
 
-export default function Navigation(props: Props) {
-  const { history } = props;
-  const links = components.map(ns => {
-    const href = history.createHref({ pathname: `/${ns}` });
-    const handleClick = ({ event }) => {
-      if (event.defaultPrevented) return;
-      if (isModifiedEvent(event) || !isLeftClickEvent(event)) return;
-      event.preventDefault();
-      history.push(`/${ns}`);
-    };
-    return (
-      <Text bold leading="tall" color="darkGray" size="lg">
-        <Link href={href} onClick={handleClick}>
-          {ns}
-        </Link>
-      </Text>
-    );
-  });
-  const options = [{ label: '-', value: '#' }].concat(
-    components.map(ns => ({
-      label: ns,
-      value: `/${ns}`,
-    }))
-  );
-  const m = window.location.hash.match(/#(\/[\w]+)/);
-
+export default function Navigation({ history }: Props) {
   return (
     <Box>
-      <Box mdDisplay="none" flex="grow">
-        <SelectList
-          id="nav"
-          onChange={({ value }) => history.push(value)}
-          options={options}
-          value={(m && m[1]) || '#'}
-        />
-      </Box>
-      <Box display="none" mdDisplay="flex" direction="column" flex="grow">
+      <Box
+        display="flex"
+        smDisplay="none"
+        overflow="scrollX"
```

Comment:
@chrislloyd Nice! Been thinking it would be great to add something to `SegmentedControl` as well, or should we always let the containing element take care of it?

![image](https://user-images.githubusercontent.com/127199/39676013-ebca9a98-5117-11e8-8fc9-1427a2ffc2b3.png)

Reply:
I think we should build out the `Tabs` component. I see SegmentedControl being more of a mis-used form element whereas Tabs are for navigation. I just didn't use them here because they look a bit ridiculous and are _very_ specific to a few pages.

Reply:
@chrislloyd makes sense.

Seems like there was some miscommunication, tried to allude to the `overflow=scrollX` specifically. I was suggesting to add something similar to `SegmentedControl` so it works better responsively. Not to use it for the main navigation.